lib/app/views/layout.erb

```diff
@@ -62,10 +62,10 @@
     <footer class="footer">
       <div class="container row">
         <div class="col-sm text-left">
-          <a target="_blank" href="https://github.com/onc-healthit/inferno">Open Source</a>
+          <a target="_blank" href="https://github.com/onc-healthit/inferno-program">Open Source</a>
         </div>
         <div class="col-sm text-center">
-          <a target="_blank" href="https://github.com/onc-healthit/inferno/issues">Issues</a>
+          <a target="_blank" href="https://github.com/onc-healthit/inferno-program/issues">Issues</a>
```

Comment:
This also needs to be fixed here:

https://github.com/onc-healthit/inferno-program/blob/d941b150e2289843f1ab1401917773c95dddde90/lib/app/views/index.erb#L66

Reply:
We use a different index page for inferno_program.  This only shows up for community.  Confusing, I know.